app/Module.java

```diff
@@ -124,7 +124,7 @@ protected void configure() {
     @Provides
     @RequestScoped
     @NavigationCategoryTree
-    private CategoryTree provideNavigationCategoryTree(final CategoryTreeConfiguration configuration, final CategoryTree categoryTree) {
+    private CategoryTree provideNavigationCategoryTree(final CategoriesSettings configuration, final CategoryTree categoryTree) {
```

Comment:
"CategorIESSettings". I had a long internal discussion whether "categories" should be in singular or plural. Reason why in plural: that's the name of the package and configuration element, and the idea is to have same name. Also it is not the configuration about one category, but all of them. But not exactly the category tree.